File: 2727-number-of-senior-citizens/2727-number-of-senior-citizens.java
class Solution {
    public int countSeniors(String[] details) {
        int age[] = new int[details.length];
        int count = 0;
        for(int i=0; i< details.length; i++){
            String temp = details[i].substring(11, 13);
            age[i] = Integer.parseInt(temp);
            if(age[i] > 60){
                count++;
            }
        }
        
        return count;
    }
}

File: 2727-number-of-senior-citizens/2727-number-of-senior-citizens.py
class Solution(object):
    def countSeniors(self, details):
        i = 0
        for p in details:
            if int(p[11:13]) > 60:
                i += 1
        return i
        """
        :type details: List[str]
        :rtype: int
        """
        